packages/medusa/src/api-v2/admin/products/validators.ts

```diff
@@ -679,3 +678,44 @@ export class ProductTypeReq {
   @IsString()
   value: string
 }
+
+export class ProductVariantPricesCreateReq {
+  @IsString()
+  currency_code: string
+
+  @IsInt()
+  amount: number
+
+  @IsOptional()
+  @IsInt()
+  min_quantity?: number
+
+  @IsOptional()
+  @IsInt()
+  max_quantity?: number
+}
+
+export class ProductVariantPricesUpdateReq {
+  @IsString()
+  @IsOptional()
+  id?: string
+
+  @ValidateIf((o) => !o.id)
+  @Validate(XorConstraint, ["currency_code"])
+  region_id?: string
```

Comment:
this should now be under "rules" object

Reply:
@riqwan removing it for now, see my comment above.